File: frontend/src/components/Pronouns.js
import React from "react";

export const PRONOUNS = [
  ["NONE", ""],
  ["HE_HIM", "he / him"],
  ["SHE_HER", "she / her"],
  ["THEY_THEM", "they / them"],
  ["OTHER", "other"],
  ["DECLINE", "decline to choose"]
];

export default ({ pronouns }) => (
  <React.Fragment>{PRONOUNS.find(p => p[0] === pronouns)[1]}</React.Fragment>
);

File: backend/atlas/schema/pronouns.py
import graphene


class Pronouns(graphene.Enum):
    NONE = ""
    HE_HIM = "HE_HIM"
    SHE_HER = "SHE_HER"
    THEY_THEM = "THEY_THEM"
    OTHER = "OTHER"
    DECLINE = "DECLINE"

    @property
    def description(self):
        if self == Pronouns.HE_HIM:
            return "he / him"
        if self == Pronouns.SHE_HER:
            return "she / her"
        if self == Pronouns.THEY_THEM:
            return "they / them"
        if self == Pronouns.OTHER:
            return "other"
        if self == Pronouns.DECLINE:
            return "decline to choose"
        return ""
